edgelet/edgelet-http/src/lib.rs

```diff
@@ -303,7 +302,7 @@ impl HyperExt for Http {
         &self,
```

Comment:
Github won't let me add a comment on the actual line, but also change the attribute on line 299 to `cfg_attr(not(unix), allow(unused_variables))`

Also move it to be on the trait method instead of on the impl.